mission_control/navigator_launch/launch/simulation.launch

```diff
@@ -38,6 +38,11 @@
         <param name="autorepeat_rate" value="5"/>
     </node>
 
+    <node pkg="navigator_emergency_control" type="navigator_emergency_control" name="navigator_emergency_xbee">
+        <param name="port" type="string" value="/dev/serial/by-id/usb-uf-mil_xbee_b267017f-if00" />
```

Comment:
Sorry for the confusing wording. What I meant to ask was if using the chip on the boat or any of our computers would give the same serial ID. It sounds like this is the case.